simulations.go

```diff
@@ -73,7 +130,7 @@ type SimulationSingleEventUpdate struct {
 	// Type: Single event sent for this simulation, in the format `entity.event_type`.
 	Type EventTypeName `json:"type,omitempty"`
 	// Payload: Simulation payload. Pass a JSON object that matches the schema for an event type to simulate a custom payload. Set to `null` to clear and populate with a demo example.
-	Payload *Payload `json:"payload,omitempty"`
+	Payload *paddlenotification.NotificationPayload `json:"payload,omitempty"`
```

Comment:
I _think_ this should not be a pointer. @davidgrayston-paddle if you could check this against the API by dropping the pointer and making a call 🙏